src/main/java/com/aws/greengrass/clientdevices/auth/metrics/MetricsEmitter.java

```diff
@@ -52,4 +70,30 @@ public void stop() {
             }
         }
     }
+
+    @SuppressWarnings("PMD.CloseResource")
+    private PutComponentMetricResponse emitMetrics() {
+        if (unauthorized) {
+            //Avoid trying to repeatedly publish when unauthorized
+            stop();
+            return new PutComponentMetricResponse();
+        }
```

Comment:
when we last talked we said not to do this. Don't do any special checking, just handle the authorization error gracefully.